src/libraries/Microsoft.PowerFx.Interpreter/ReflectionFunction.cs

```diff
@@ -336,19 +337,44 @@ public async Task<FormulaValue> InvokeAsync(IServiceProvider serviceProvider, Fo
                 }
             }
 
-            foreach (var arg in args)
+            List<ErrorValue> errors = null;
+            for (var i = 0; i < args.Length; i++)
             {
-                if (arg is LambdaFormulaValue lambda)
+                if (args[i] is ErrorValue ev)
+                {
+                    if (errors == null)
+                    {
+                        errors = new List<ErrorValue>();
+                    }
+
+                    errors.Add(ev);
+                }
+                else if (args[i] is BlankValue && _info.ParamTypes[i] is NumberType)
+                {
+                    args[i] = FormulaValue.New(0);
+                    args2.Add(args[i]);
+                }
+                else if (args[i] is BlankValue && _info.ParamTypes[i] is StringType)
+                {
+                    args[i] = FormulaValue.New(string.Empty);
+                    args2.Add(args[i]);
+                }
+                else if (args[i] is LambdaFormulaValue lambda)
                 {
                     Func<Task<BooleanValue>> argLambda = async () => (BooleanValue)await lambda.EvalAsync();
                     args2.Add(argLambda);
                 }
                 else
                 {
-                    args2.Add(arg);
+                    args2.Add(args[i]);
                 }
             }
 
+            if (errors != null)
+            {
+                return ErrorValue.Combine(IRContext.NotInSource(FormulaType.BindingError), errors);
```

Comment:
>FormulaType.BindingError [](http://example.com/codeflow?start=64&length=24)

This should be the function's return Type. 

Errors can be typed. 
Ie, Abs(1/0) returns an error of type number.